nifi-extension-bundles/nifi-airtable-bundle/nifi-airtable-processors/src/main/java/org/apache/nifi/processors/airtable/QueryAirtableTable.java

```diff
@@ -108,10 +109,10 @@ public class QueryAirtableTable extends AbstractProcessor {
             .required(true)
             .build();
 
-    static final PropertyDescriptor API_KEY = new PropertyDescriptor.Builder()
-            .name("api-key")
-            .displayName("API Key")
-            .description("The REST API key to use in queries. Should be generated on Airtable's account page.")
+    // API Keys are deprecated, Airtable now provides Personal Access Tokens instead.
+    static final PropertyDescriptor PAT = new PropertyDescriptor.Builder()
+            .name("pat")
```

Comment:
The display name has been lost after the first commit.
```suggestion
            .name("pat")
            .displayName("Personal Access Token")
```